tools/buddy-opt/buddy-opt.cpp

```diff
@@ -78,6 +84,7 @@ int main(int argc, char **argv) {
   // Register all MLIR passes.
   mlir::registerAllPasses();
   mlir::buddy::registerPointwiseConvToGemmPass();
+  // mlir::buddy::registerPointwiseConvToGemmForNhwcFhwcPass();
```

Comment:
Remove it since `PointwiseConvToGemmForNhwcFhwcPass` has been registered in line55.